shaders/src/common_lighting.fs

```diff
@@ -71,6 +75,64 @@ float computeMicroShadowing(float NoL, float visibility) {
     return microShadow * microShadow;
 }
 
+/**
+ * IBL utilities
+ */
+
+vec2 IntersectAABB(vec3 rayOrigin, vec3 rayDir, vec3 boxMin, vec3 boxMax) {
```

Comment:
We don't use upper case as the first letter of functions in our code style.

Reply:
I'm amending mine (6dc6210f556cee2f8ce36cfd404588c5ddb3fa2d) but why are `Irradiance_SphericalHarmonics`, `Irradiance_RoughnessOne`, and `PrefilteredDFG_LUT` cased differently? Is it OK if I fix up those too to start with lower-case or camel-cased functions are to be handled and cased differently? If so, what does the prefix/postfix signify?